packages/devextreme-scss/scss/widgets/base/chat/layout/chat-messagelist/_index.scss

```diff
@@ -9,6 +9,14 @@
     flex-direction: column;
     width: 100%;
   }
+
+  .dx-scrollable-native {
+    &.dx-scrollable-native-ios{
+      .dx-scrollable-content {
+        min-height: 100%;
```

Comment:
Let's create a tech debt card for reconsidering this style on scrollable's level. I don't like we need override it in components where we use scrollable

Reply:
I unarchived the card in our Backlog and added this case there. It's attached to our discussion thread in Teams.